indicators/mygovau-7d9649.yml

```diff
@@ -0,0 +1,23 @@
+title: MyGovAU Phishing Kit - 7d9649
+
+description: |
+	Detects a phishing kit using the MyGov Australia branding.
+
+references: 
+	- https://urlscan.io/result/d0a5f9fd-b960-47a5-8b44-3f9c7ffc9cd8/
+	
+	
+detection:
+    chromeHTMLFragments:
```

Comment:
Also, I think the killbot API key (`Gwhrue6nv7INVxMChew6XgpGD3_aLJ3imwX5OnvbETNpA`) left by the threat actor should be a detection string as it has a high entropy value 

I also think that the id (`00NF000000DAKs6`) of the input field where the IP of the visitor is stored, would be a good candidate for a detection string

The CSRF token may also be of use `31f75120-71a5-4089-8840-1c95d0e3443d` as it also has high entropy